facebook/micro/monthly-archive.sh

```diff
@@ -8,13 +8,27 @@ else
 fi
 echo ${MONTH}
 R_MONTH=${MONTH#*_}; R_MONTH=${R_MONTH#0}
-BATCH="cd fb-public-results\nls -1 cvid_responses_${MONTH}*.gz"
-sftp -b <(echo -e "${BATCH}") -P 2222 fb-automation@ftp.delphi.cmu.edu 2>/dev/null | \
-    grep "^cvid" | \
-    awk -F_ 'BEGIN{print "cd fb-public-results"} {key=$3 $4 $5; if (key!=last && last!="") {print record} last=key; record=$0} END{print record}' | \
-    sed '/^cvid/ s/^/get /' >fetch.sftp
-sftp -b fetch.sftp -P 2222 fb-automation@ftp.delphi.cmu.edu
-OUT=${MONTH/_/-}
-Rscript ../monthly-files.R ${MONTH%_*} ${R_MONTH} . >${OUT}.csv
-gzip ${OUT}.csv
-sftp -b <(echo -e "cd fb-public-results\nput ${OUT}.csv.gz") -P 2222 fb-automation@ftp.delphi.cmu.edu
+
+# Save script's arguments to reference by name in perform_rollup_and_post() so the script's
+# positional arguments and perform_rollup_and_post's positional arguments don't get confusing.
+arg0=$0
+arg3=$3
+arg4=$4
+arg5=$5
```

Comment:
ah, monthly-archive.sh doesn't have any positional arguments. The $3 $4 $5 in the awk script refer to the third, fourth, and fifth words in the line being parsed from the output of `ls -1 covid_responses...`. The single quote prevents the shell from substituting the $-expressions at execution time, instead preserving them as-is for awk to interpret.